lib/flutter_redux.dart

```diff
@@ -471,7 +471,7 @@ class _StoreStreamListenerState<S, ViewModel>
     _computeLatestValue();
 
     if (widget.onInitialBuild != null) {
-      WidgetsBinding.instance?.addPostFrameCallback((_) {
```

Comment:
On the `stable` channel this does not compile, works fine on `beta`.

Reply:
Ah woops haha, I was developing against `master` branch :P I'll revert this part.